tensorboard/webapp/widgets/line_chart_v2/lib/worker/compact_data_series_test.ts

```diff
@@ -104,49 +104,33 @@ describe('line_chart_v2/lib/compact_data_series', () => {
       );
     });
 
-    it('rounds to float 32', () => {
+    it('transfers without losing precision', () => {
       const dataSeries = [
         {
           id: 'foo',
-          points: [{x: 1, y: Number.MIN_VALUE}],
+          points: [
+            {x: 1, y: Number.MIN_VALUE},
+            {x: 1, y: 0.3},
+            {x: 1, y: 1e-10},
+          ],
         },
       ];
 
       const actual = decompactDataSeries(compactDataSeries(dataSeries));
-      expect(actual).not.toEqual(dataSeries);
-      expect(actual).toEqual([
-        {
-          id: 'foo',
-          points: [{x: 1, y: 0}],
-        },
-      ]);
+      expect(actual).toEqual(dataSeries);
     });
 
-    it('does not round fractiional numbers', () => {
+    it('transfers date', () => {
+      const time = new Date('2020-01-01').getTime();
```

Comment:
Cool; looks like the imprecision for realistic dates is about 3 minutes,
since `f32::EPSILON * 1577836800_000.0 == 188092.8`. Change LGTM.